### PR TITLE
Ignore negative nb argument on splice to fix #277

### LINKED ISSUE #277: Is it possible to replace an item in an array?
I guess the way to do it is with method 'splice' but:

Let's say I want to change the content of element at index 0. I want to replace banana with cherry:

```javascript
var tree = new Baobab({
  list: ['banana', 'apple']
});

// It works!
tree.select('list').splice([0, -1, 'cherry']);
```
But if I have only one element in the array:

```javascript
var tree = new Baobab({
  list: ['banana']
});

// It does not work! I get: ['cherry', 'banana']
tree.select('list').splice([0, -1, 'cherry']);
```

It looks as if, in that case, the item is added to the beginning instead of being replaced. It's possible that I didn't get my splice parametres right but I didn't find help on that.

### PR DESCRIPTION
**Pull request by SaphuA**
Fixes #277

### CODE REVIEW

**Comment by Yomguithereal** [comment]
Thanks @SaphuA.

File: test/suites/helpers.js [hunk 1]
```diff
@@ -158,6 +158,14 @@ describe('Helpers', function() {
         ['yellow', 'gold', 'purple']
       );
     });
+	
+    it('should treat a negative nb argument as 0.', function() {
+      var array = ['yellow', 'blue', 'purple'];
```
Comment (Yomguithereal) [inline test/suites/helpers.js]: Can I ask you to turn this into a `const` please?

Comment (SaphuA) [inline test/suites/helpers.js]: Done!

**Comment by Yomguithereal** [comment]
Thanks @SaphuA. Merging right now.

**Pull request merged**